components/x-teaser-timeline/__tests__/lib/transform.test.js

```diff
@@ -298,5 +298,55 @@ describe('buildModel', () => {
 				groupedItems[2]
 			])
 		})
+		test('returns correct model for multiple custom slots', () => {
+			const result = buildModel({
+				items,
+				timezoneOffset: 0,
+				localTodayDate: '2020-01-14',
+				customSlotContent: [{ foo: 1 }, { bar: 2 }],
+				customSlotPosition: [0, 3]
+			})
+			expect(result).toEqual([
+				{
+					...groupedItems[0],
+					items: [
+						{ foo: 1 },
+						groupedItems[0].items[0],
+						groupedItems[0].items[1],
+						groupedItems[0].items[2],
+						{ bar: 2 },
```

Comment:
That's index 4. If I read the input properly it would expect it at index 3?

Reply:
Reading the code, I understand that it leaves 3 items between the first insert position and the next custom slot. I think the "Position" variable name is a bit confusing in that case.

Reply:
The confusing part is that the insertion is performed _simultaneously_. It's not leaving a 3 offset, but it's inserting at positions 0 and 3 before the change.
ie. it inserts something before `groupedItems[0].items[0]` and before `groupedItems[0].items[3]`
By adding another test case (positions 2 and 3), I'm hoping that makes it clearer.

I think this is preferred because the `customSlot` addition is for non-news items being inserted, so we wouldn't want to have to factor in that added slot in the decision of adding another slot.